strategy/json/sql/postgres/single_stream_strategy.go

```diff
@@ -43,32 +45,29 @@ func (s *SingleStreamStrategy) CreateSchema(tableName string) []string {
     event_id UUID NOT NULL,
     event_name VARCHAR(100) NOT NULL,
     payload JSON NOT NULL,
-    metadata JSONB NOT NULL,
+    metadata JSONB NOT NULL, 
+    aggregate_type VARCHAR(50) NOT NULL,
```

Comment:
Is varchar 50 enough? Could we perhaps consider 100 at least?

Reply:
This is just the default, implementations can write their own migration files, overwriting this.